cv-data-controller/src/test/java/com/trihydro/cvdatacontroller/controller/ActiveTimControllerTest.java

```diff
@@ -1228,18 +1275,16 @@ public void ResetExpirationDate_SUCCESS_Over_500_Records() throws SQLException {
         }
         updateTwo = updateOne.substring(0, updateOne.length() - 1);
         updateTwo += ")";
-        verify(mockConnection)
-                .prepareStatement(updateOne);
-        verify(mockConnection)
-                .prepareStatement(updateTwo);
+        verify(mockConnection).prepareStatement(updateOne);
+        verify(mockConnection).prepareStatement(updateTwo);
 
         for (int i = 0; i < 500; i++) {
             verify(mockPreparedStatement).setLong(i + 1, (Long.valueOf(i)));
         }
-        for(int i = 500; i < 600; i++){
+        for (int i = 500; i < 600; i++) {
             verify(mockPreparedStatement).setLong(i - 499, (Long.valueOf(i)));
         }
-        verify(mockPreparedStatement).close();
+        verify(mockPreparedStatement, times(2)).close();

```

Comment:
Note: After refactoring to use try-with-resources, this test began failing due to multiple prepared statements being created and subsequently closed. This suggests that some prepared statements were not being properly closed before the refactor. To address this, the verification statement was updated to expect two instances of prepared statement closures.